src/app/static/package.json

```diff
@@ -105,12 +105,15 @@
     "webpack-node-externals": "^3.0.0"
   },
   "dependencies": {
+    "@m-kusumgar/vue3-treeselect": "^1.0.18",
```

Comment:
We can change this to reside's npm account, I just didn't have the details so I just made my own npm account. The treeselect component was throwing a couple of warnings like $createElement was unused or Transition tag not recognised, little things in the source code so I changed the source code. Note, I have not changed the functionality, just removed unused variables from the output or replaced a couple transition tags with divs, that is all I did to remove the warnings

Reply:
Could this be changed in this PR?

Reply:
@M-Kusumgar If you send me your npm user name I'll add you to the reside-ic organisation. Is it just `m-kusumgar`?

Reply:
Yep I believe it is just m-kusumgar